proposals/lip-0052.md

```diff
@@ -1840,11 +1844,15 @@ def transferCrossChain(
 ) -> None:
 
     if receivingChainID == OWN_CHAIN_ID:
-        emitFailedTransferCrossChainEvent(senderAddress, nftID, amount, receivingChainID, recipientAddress, INVALID_RECEIVING_CHAIN)
+        emitFailedTransferCrossChainEvent(senderAddress, recipientAddress, nftID, receivingChainID, INVALID_RECEIVING_CHAIN)
```

Comment:
This check is superfluous now

Reply:
As discussed, it's not wrong to keep this check for reasons of clarity.